packages/grid/x-data-grid/src/hooks/features/density/densitySelector.ts

```diff
@@ -18,17 +18,7 @@ export const gridDensityHeaderHeightSelector = createSelector(
   (density) => density.headerHeight,
 );
 
-export const gridDensityHeaderGroupingMaxDepthSelector = createSelector(
```

Comment:
Nice for removing these - it doesn't make much sense for them to be in the density selectors file